src/components/form/day-picker/index.tsx

```diff
@@ -115,12 +122,12 @@ export const DayPicker = ({
 
   React.useEffect(() => {
     const close = (e) => {
-      if(e.keyCode === 27 || e.keyCode === 13){
+      if (e.keyCode === 27) {
```

Comment:
I'd write a comment with a definition of the key, which code is 27, as it's a bit cryptic

```typescript
// key with code 27 is "Escape"
if (e.keyCode === 27){
closeCalendar();
}
```